ydb/core/sys_view/ut_kqp.cpp

```diff
@@ -114,6 +114,33 @@ void CreateRootTable(TTestEnv& env, ui64 partitionCount = 1, bool fillTable = fa
     }
 }
 
+void SetupAuthEnvironment(TTestEnv& env) {
+    env.GetServer().GetRuntime()->SetLogPriority(NKikimrServices::FLAT_TX_SCHEMESHARD, NLog::PRI_DEBUG);
+    env.GetServer().GetRuntime()->SetLogPriority(NKikimrServices::SYSTEM_VIEWS, NLog::PRI_TRACE);
+    CreateTenantsAndTables(env, true);
+}
+
+void SetupAuthAccessEnvironment(TTestEnv& env) {
+    env.GetServer().GetRuntime()->SetLogPriority(NKikimrServices::FLAT_TX_SCHEMESHARD, NLog::PRI_DEBUG);
+    env.GetServer().GetRuntime()->SetLogPriority(NKikimrServices::SYSTEM_VIEWS, NLog::PRI_TRACE);
+    env.GetServer().GetRuntime()->GetAppData().AdministrationAllowedSIDs.emplace_back("root@builtin");
+    env.GetServer().GetRuntime()->GetAppData().AdministrationAllowedSIDs.emplace_back("user1");
```

Comment:
Предлагаю явно обозначить в имени, что это не обычный пользователь. Тогда тесты будут проще читаться.
```suggestion
    env.GetServer().GetRuntime()->GetAppData().AdministrationAllowedSIDs.emplace_back("root-admin");
```

Reply:
С однотипными именами проще проверять всякие сортировки, я бы так оставил

К тому же там в тест-кейзах явно написано в блоке

```
 // user1 has /Root GenericUse access
```

https://github.com/ydb-platform/ydb/pull/13781/files#diff-a7ae7f2ec7888e896770172a7dda539285c781007c9df7f8b3fc012a49cd4876R2777

Reply:
Лучше все же придумать какое-то имя, чтобы было понятно будущим поколениям, что это админ.
И при этом можно сделать таким, чтобы сортировка удобная была.